packages/experiments/src/components/Shimmer/examples/Shimmer.Application.Example.tsx

```diff
@@ -7,15 +7,23 @@ import {
   IExpandingCardProps
 } from 'office-ui-fabric-react/lib/HoverCard';
 import { createListItems } from '@uifabric/example-app-base';
-import './Shimmer.Example.scss';
+// import {
```

Comment:
undo this

Reply:
@cschleiden by undo you mean remove the commented import?

Reply:
yep :)